### PR TITLE
Provide an mechanism to disable managed cluster alert forwarding

### PR DESCRIPTION
**Pull request by ngraham20**
Implements features:
https://issues.redhat.com/browse/OCPBUGS-1025
https://issues.redhat.com/browse/ACM-1753

### CODE REVIEW

**Review by marcolan018 — APPROVED** [review]
lgtm

File: operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go [hunk 1]
```diff
@@ -192,8 +192,12 @@
...
 
 	// create or update the cluster-monitoring-config configmap and relevant resources
-	if err := createOrUpdateClusterMonitoringConfig(ctx, hubInfo, clusterID, r.Client, installPrometheus); err != nil {
-		return ctrl.Result{}, err
+	if hubInfo.AlertmanagerEndpoint != "" {
+		if err := createOrUpdateClusterMonitoringConfig(ctx, hubInfo, clusterID, r.Client, installPrometheus); err != nil {
+			return ctrl.Result{}, err
+		}
+	} else {
+		revertClusterMonitoringConfig(ctx, r.Client, installPrometheus)
```
[the remark this answers is not in the document]
Comment (subbarao-meduri) [reply operators/endpointmetrics/controllers/observabilityendpoint/observabilityaddon_controller.go]: fixed in latest commit

**Pull request closed without merging**